build/yamls/audit-webhook.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: audit-pv-volume
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi 
+  accessModes:
+    - ReadWriteMany
+  hostPath:
```

Comment:
add a comment to indicate that using hostPath here is for testing purposes only.

BTW I assume this is broken on a multiple-Node cluster if the Pod is re-scheduled on a different Node?

Also do not use `"/home/vagrant/audit"` but something more generic like `"/data/antrea-audit"`

Reply:
It shouldn't break because ReadWriteMany allows multiple nodes to write to a Persistent Volume. I'll test this more and update.

Reply:
What I meant is the the storage will be local to a Node. So if the Pod is re-scheduled and moved to another Node, the previous repo will be lost.

See the documentation: https://kubernetes.io/docs/concepts/storage/persistent-volumes/

> hostPath - HostPath volume (for single node testing only; WILL NOT WORK in a multi-node cluster; consider using local volume instead)

I actually think we should switch to `local` as suggested, but you should test it to make sure it works

see also https://kubernetes.io/docs/concepts/storage/volumes/#local

Reply:
Sounds good, I'll switch to that.